package.json

```diff
@@ -23,7 +23,9 @@
     "lucide-react": "^0.435.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-emoji-render": "^2.0.1",
     "react-hook-form": "^7.53.0",
+    "react-icons": "^5.3.0",
```

Comment:
If these are not being used, please revert the changes and the `package-lock.json`

Reply:
Okk cool